assets/scss/print.scss

```diff
@@ -5,7 +5,8 @@
 
 .govuk-hint,
 .govuk-tag,
-.govuk-caption-l {
+.govuk-caption-l,
+.govuk-details__summary {
   color: #0b0c0c
```

Comment:
LGTM; out of interest, could you use the variable`$govuk-text-colour` instead of the hex reference?
https://design-system.service.gov.uk/styles/colour/

Reply:
We didn't previously have access to the colour variables in the `print.scss`, I've included the relevant imports we need from `govuk` (not all components styles just the base/core) to allow us to use variables here.